tests/Operation/AggregateFunctionalTest.php

```diff
@@ -14,6 +14,25 @@
 
 class AggregateFunctionalTest extends FunctionalTestCase
 {
+    public function testBatchSizeIsIgnoredIfPipelineIncludesOutStage()
+    {
+        (new CommandObserver)->observe(
+            function() {
+                $operation = new Aggregate(
+                    $this->getDatabaseName(),
+                    $this->getCollectionName(),
+                    [['$out' => $this->getCollectionName() . '.output']],
```

Comment:
Shouldn't this new collection be drop'ed in the tearDown?

Reply:
Added a drop later in the test, along with another test method that was leaving behind an output collection.